command/agent/auth/auth.go

```diff
@@ -371,7 +395,7 @@ func (b *agentBackoff) next() {
 	b.current = maxBackoff - time.Duration(trim)
 }
 
-func (b *agentBackoff) reset() {
+func (b *agentBackoff) reset(initialBackoff time.Duration) {
 	b.current = initialBackoff
```

Comment:
If you add (and set) an `initial` private field on `agentBackoff`, reset can remain without args and `b.current` can be set to `b.initial` like so:

```suggestion
func (b *agentBackoff) reset() {
	b.current = b.initial
```

Reply:
Good call out, I will update this on my end.